backend/lib/endpoints/posts.js

```diff
@@ -98,6 +100,52 @@ async function routes(app) {
       return updatedPost;
     },
   );
+
+  app.put(
+    "/:postId/likes/:userId",
+    { preValidation: [app.authenticate], schema: likeUnlikePostSchema },
+    async (req) => {
+      const { postId, userId } = req.params;
+      // todo: get user id from JWT
```

Comment:
with passport I think you can get that from `req.user.id` for the current user. or did we remove passport?

Reply:
Yep that’s right, we had that with express but for Fastify we now have a more custom auth middleware which closely resembles passport. Still need to check if the auth is fully functional... wasn’t completely working last time i tried.